state/controller.go

```diff
@@ -166,18 +166,17 @@ func (st *State) checkSpaceIsAvailableToAllControllers(spaceName string) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
+	netSpace, err := space.NetworkSpace()
```

Comment:
Just moved it out of the loop so that the subnets are materialised the one time.